Revamp docs directory and add more docs (#21)
ScratchAddons/website-v2: content/docs/reference/addon-api/addon.notifications.md

---
title: addon.account
description: Allows addons to create browser notifications. This API slightly differs from the browser.notifications API extensions can use.
weight: 6
---

**Available in persistent scripts.**  
**NOT available in userscripts.**  
**Required permissions: `notifications`**

Allows addons to create browser notifications. This API slightly differs from the `browser.notifications` API extensions can use.

## Functions
### addon.notifications.create(optionsObject)
Returns a promise that resolves to the ID (string) of the created notification.  
Shows a notification to the user according to the options object.  
[Options object reference](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/notifications/NotificationOptions).  

> Note: for your convenience, neither specifying `buttons` and `requireInteraction` throws on Firefox.

Reviewer: Unclear, maybe this?
```suggestion
Note: for your convenience, specifying `buttons` or `requireInteraction` throws on Firefox.
```
